Containers/talk/Dockerfile

```diff
@@ -1,18 +1,18 @@
 FROM nats:2.9.18-scratch as nats
 FROM strukturag/nextcloud-spreed-signaling:1.1.2 as signaling
-FROM coturn/coturn:4.6.2-r3-alpine
-USER root
-# Pin alpine version manually as long as https://github.com/coturn/coturn/issues/1226 is not done
-ENV ALPINE_VERSION=3.18
+FROM ghcr.io/processone/eturnal:x.x.x-otpxx AS eturnal
+FROM alpine:3.18.2
 
-COPY --from=nats /nats-server /usr/local/bin/nats-server
+ENV STUN_SERVICE="stun.nextcloud.com 443"
```

Comment:
Apparently we dont need this anymore with https://github.com/nextcloud/all-in-one/pull/2855#discussion_r1241233958

```suggestion
```

Reply:
https://github.com/nextcloud/all-in-one/pull/2855/files#r1241234554

Reply:
If you want to fallback to STUN, the variable is required. eturnal needs to know which STUN server to query.

Reply:
> If you want to fallback to STUN, the variable is required. eturnal needs to know which STUN server to query.

true, thats why I resolved the comment in the meantime :)
https://github.com/nextcloud/all-in-one/pull/2855/files#r1241234554